network-area-diagram/src/main/java/com/powsybl/nad/svg/iidm/DefaultLabelProvider.java

```diff
@@ -149,4 +150,13 @@ protected SvgParameters getSvgParameters() {
     protected ValueFormatter getValueFormatter() {
         return this.valueFormatter;
     }
+
+    public boolean isDisplayAngle() {
```

Comment:
There's no need to make it available to the outside, you can remove this and access directly to the value when needed